### PR TITLE
chore: Updates for rust 1.80

### PR DESCRIPTION
**Pull request by jrconlin**
* Mostly documentation fixes.
* fix broken `cfg!` call
* Add docs for StubRouter

### CODE REVIEW

File: autoendpoint/src/routers/stub/settings.rs [hunk 1]
```diff
@@ -8,7 +13,8 @@ pub struct StubSettings {
     pub server_credentials: String,
 }
 
-/// Credential information for each application
+/// StubServerSettings contains the default available client types.
+///
```
Comment (taddes) [inline autoendpoint/src/routers/stub/settings.rs:17]: Think we can remove this redundant`///` 
```suggestion
```

File: autoendpoint/src/routers/stub/router.rs [hunk 2]
```diff
@@ -31,10 +48,12 @@ impl StubRouter {
         })
     }
 
-    /// Create Test clients for each application
+    /// Create Test clients for each application. Tests can specify which client
+    /// to use by designating the value in the `app_id` of the subscription URL. (e.g.)
```
Comment (taddes) [inline autoendpoint/src/routers/stub/router.rs:52]: Was there supposed to be an example added after the description, hence the ` (e.g.)` ?

File: autoendpoint/Cargo.toml [hunk 3]
```diff
@@ -69,6 +69,14 @@ tokio = { workspace = true, features = ["fs", "macros"] }
...
 [features]
 default = ["bigtable"]
+# data store types
 bigtable = ["autopush_common/bigtable"]
+
+# enable emulator to call locally run data store.
 emulator = ["bigtable"]
+
+# Enable "stub" router for local testing purposes.
+# The "stub" will return errors and
```
Comment (taddes) [inline autoendpoint/Cargo.toml:79]: Is there a missing description after the and that we need here?

**Review by taddes — CHANGES_REQUESTED** [review]
Thanks for the update! Just some notes on a lonely `///` and a few missing comments/examples.

**Comment by jrconlin** [comment]
Sorry, this was meant to be a draft until the docker images for 1.80 land tomorrow.

**Review by taddes — APPROVED** [review]
🚀 Tyty!

**Pull request merged**